ang/civicase/CaseDetailsFileTab.html

```diff
@@ -1,13 +1,13 @@
+<div crm-ui-debug="getfiles"></div>
 <div civicase-util="caseutil"></div>
 <div civicase-api3-ctrl="getfiles"
      civicase-api3="['Case','getfiles', {case_id: item.id, text: '', options: {xref:1, limit:0}}]"
      on-refresh="caseutil.formatActivities(result.xref.activity)">
 </div>
-
 <div civicase-file-uploader="item" on-upload="getfiles.refresh()"></div>
-
-<div crm-ui-debug="getfiles"></div>
-<div class="panel panel-default act-feed-panel">
-  <div class="panel-header" civicase-file-filter="getfiles"></div>
-  <div class="panel-body clearfix" civicase-file-list="getfiles"></div>
+<div class="civicase__file-tab-actions">
+    <!--    For Bulk Actions
+   <div class="civicase__file-filter-actions"></div> -->
```

Comment:
This will be used for implementing bulk actions. This div will contains bulk actions HTML. This has been placed here to give the next developer idea where he/she needs to place the HTML.